src/main/java/net/snowflake/client/jdbc/cloud/storage/SnowflakeS3Client.java

```diff
@@ -237,7 +237,7 @@ private void setupSnowflakeS3Client(
   }
 
   static String getDomainSuffixForRegionalUrl(String regionName) {
-    return regionName.startsWith("cn-") ? "amazonaws.com.cn" : "amazonaws.com";
+    return regionName.toLowerCase().startsWith("cn-") ? "amazonaws.com.cn" : "amazonaws.com";
```

Comment:
dumb question, how is this `getDomainSuffixForRegionalUrl` method used in JDBC?

Reply:
when we have a region and we should use regional URL for S3 we call this method to know the domain:https://github.com/snowflakedb/snowflake-jdbc/blob/823cecc40d7578b2aa14bb3418cf55c1339e491b/src/main/java/net/snowflake/client/jdbc/cloud/storage/SnowflakeS3Client.java#L223-L231